.travis.yml

```diff
@@ -6,42 +6,26 @@ cache:
   directories:
     - $HOME/.composer/cache/files
 
-php: [5.3, 5.4, 5.5, 5.6, 7.0, 7.1, hhvm]
+php: [7.1]
```

Comment:
dropping CI for all older releases is a no-go, especially when composer.json does not drop support for them